test/functional/fixtures/api/raw/wait/test.js

```diff
@@ -0,0 +1,31 @@
+var expect = require('chai').expect;
+
+describe.only('[Raw API] Wait action', function () {
```

Comment:
Since we have two different commands (`wait` on server and `waitForElement` on client) I propose to call this tests `/raw/wait-for-element/...`
And we need a simple test for `wait` command